test/pkgs/Dynamo Samples/pkg.json

```diff
@@ -1,12 +1,21 @@
-{"file_hash":null,
-"name":"Dynamo Samples",
-"version":"2.0.0",
-"description":"Dynamo code samples.",
-"group":"",
-"keywords":["dynamo","samples"],
-"dependencies":[],
-"license":"MIT",
-"contents":"",
-"engine_version":"2.0.0",
-"engine_metadata":"",
-"engine":"dynamo"}
\ No newline at end of file
+{
+    "file_hash": null,
+    "name": "Dynamo Samples",
+    "version": "2.0.0",
+    "description": "Dynamo code samples.",
+    "group": "",
+    "keywords": [
+        "dynamo",
+        "samples"
+    ],
+    "dependencies": [],
+    "license": "MIT",
+    "contents": "",
+    "engine_version": "2.0.0",
+    "engine_metadata": "",
+    "engine": "dynamo",
+    "node_libraries": [
```

Comment:
hmm, were these changes necessary to get the package command to work?

Reply:
If so, would you mind also making the same change to the DynamoSamples repo? If we regen the sample package and then tests start failing it's going to be hard to figure out why.

Reply:
So the pkg json did not specify any node libraries, which I actually thought was required. So the package command would not know which assemblies to scan

Reply:
right - makes sense - there is some fallback behavior in dynamo for very old packages before `node_libraries` existed.